client/metric.go

```diff
@@ -0,0 +1,74 @@
+package client
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/drand/drand/chain"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func newMetricController(chainInfo *chain.Info, b PrometheusBridge) *metricController {
```

Comment:
more of this, ideally, can live in the `/metrics` package.

Can we have all the gauge / counters / etc. living there, and have a client gatherer method, parallel to how that package already provides a subset of metrics relevant to the group.

generic push code could also live in that package.